docs/migrating-v1alpha1-to-v1beta1.md

```diff
@@ -22,13 +22,13 @@ In Tekton `v1beta`, the following fields have been changed:
 | --------- | ----------|
 | `spec.inputs.params` | [`spec.params`](#changes-to-input-parameters) |
 | `spec.inputs` | Removed from `Tasks` |
-| `spec.outputs` | Removed from `Tasks` | 
+| `spec.outputs` | Removed from `Tasks` |
 | `spec.inputs.resources` | [`spec.resources.inputs`](#changes-to-pipelineresources) |
 | `spec.outputs.resources` | [`spec.resources.outputs`](#changes-to-pipelineresources) |
 
 ## Changes to input parameters
 
-In Tekton `v1beta`, input parameters have been moved from `spec.inputs.params` to `spec.params`. 
+In Tekton `v1beta`, input parameters have been moved from `spec.inputs.params` to `spec.params`.
```

Comment:
Should this be `v1beta1` instead of `v1beta`?

Reply:
It should indeed, good catch!